src/app/user-data.ts

```diff
@@ -0,0 +1,51 @@
+export class UserData {
```

Comment:
io questa onestamente la metterei come nested public class dentro a User

Reply:
Pensavamo fosse più facile da testare così con due classi definite separatamente però ci sta

Reply:
più che altro perché dubito compaia mai un userdata senza uno user

Reply:
> più che altro perché dubito compaia mai un userdata senza uno user

No infatti, ma costruire componenti modulari rende il testing molto più laborioso, per non parlare dell'includere i componenti che implementano il controllo dell'errore client-side (_vedi email e password_) in altri componenti, farli funzionare è stato un incubo e non volevamo sprecare troppo tempo

Reply:
ok, è ragionevole sotto quest'ottica. comunque cerchiamo di tenere la radice dell'app più pulita possibile. tipo io user + userdata li metterei in sottocartella classes/users/user

Reply:
@RiccardoCestaro visto che stai sistemando set e get in queste classi vedi se riesci a inserirli in due cartelle distinte e aggiornare i path degli import

Reply:
si occhio che siamo più liberi con le classi rispetto alle componenti; nel senso che 1 componente = 1 cartella ma volendo possiamo fare 1 cartella = n classi

addirittura, finchè le classi son poche le potremmo mettere tutte in una cartella "classes"

Reply:
> aggiornare i path degli import

vscode lo fa da solo questo